authorization/storage_authorization.go

```diff
@@ -37,6 +60,14 @@ func encodeAuthorization(a *influxdb.Authorization) ([]byte, error) {
 		}
 	}
 
+	// Redact Token, if needed. This is done at the lowest level so it is impossible to serialize
+	// raw tokens if hashing is enabled.
+	if s.useHashedTokens {
+		// Redact a copy, not the original. The raw Token value is still needed by the caller in some cases.
```

Comment:
Nice thank you for the comment, makes the code below much easier to reason about.